_episodes/04-lists.md

```diff
@@ -160,32 +160,42 @@ does not.
 > ## Nested Lists
 > Since a list can contain any Python variables, it can even contain other lists.
 >
-> For example, we could represent the products in the shelves of a small grocery shop:
+> For example, you could represent the products on the shelves of a small grocery shop
+> as a nested list called x:
+>
+> [![x is represented as a shelf full of produce. There are four rows of vegetables
+> on the shelf, and each row contains five baskets of vegetables. We can label
+> each basket according to the type of vegetable it contains, so the top row
+> contains (from left to right) lettuce, lettuce, peppers, zucchini, squash,
+> and basil.](../fig/04_groceries_x.png)]
+>
+> To store the contents of the shelf in a nested list, you write it this way:
 >
 > ~~~
-> x = [['pepper', 'zucchini', 'onion'],
->      ['cabbage', 'lettuce', 'garlic'],
->      ['apple', 'pear', 'banana']]
+> x = [['lettuce', 'lettuce', 'peppers', 'zucchini', 'squash', 'basil'],
+>      ['lettuce', 'lettuce', 'peppers', 'zucchini', 'peppers', 'parsley'],
+>      ['lettuce', 'cilantro', 'peppers', 'zucchini', 'squash', 'spinach'],
+>      ['cabbage', 'broccoli', 'asparagus', 'zucchini', 'squash', 'cauliflower']]
 > ~~~
 > {: .language-python}
```

Comment:
I really like the changes you have made to this callout @kris-joseph .  To me it clarifies the nested list indexing with a visual that aligns with the text. One concern I have is that this example has more than doubled the typing to create the nested list, which when taken in the context of doing the full lesson, I think can be tedious for learners. What do you think about cropping the images, so there are only 3 baskets per row? ( I would wait to see if @maxim-belkin has other suggestions to the PR before making changes)

Reply:
My only request is that the number of rows is different from the number of baskets in a row (e.g. 3 rows, 4 baskets per row). I think the suggestion to use 3 rows is a good one because instructors will be able to refer to them as top, middle, and bottom rows.